temporalio/client.py

```diff
@@ -1074,6 +1089,7 @@ def __init__(
         result_run_id: Optional[str] = None,
         first_execution_run_id: Optional[str] = None,
         result_type: Optional[Type] = None,
+        eager_start_active: bool = False,
```

Comment:
Hrmm, I'm not sure this should be on the handle. Usually the only things on a handle are things that have actual usage value for the handle regardless of whether it was created via start _or_ whether the handle is created via `get_workflow_handle` _or_ whether this constructor is called manually. This has only read value and only in one case of handle creation.

We don't tell a user on their activity handle that their activity was eagerly started either and that has seemed ok to everyone. I think we can use a metric or something else to tell a user (and a test) that workflows were started eagerly. I think we can punt on reporting this to users via the handle at first and maybe revisit when a user requests it.

Reply:
Now that I think about it, I would be ok if this was kept private and not put not on the constructor and changed to "eagerly started". So:

* Remove this constructor param
* Change init code to be `self._eagerly_started = False`
* Change property name to be `def eagerly_started(self) -> bool:` and remove docstring about cannot be mutated (implied for read-only property)
* Set the attribute post-construction in the workflow starter code

I could also be convinced of "started eagerly" instead of "eagerly started" though I like the latter because it matches the parameter name from start/execute.

Reply:
I would like to have the discussion of how we may expose in Go and Java before exposing in Python. If we find our two most used SDKs haven't/don't need to expose that they started eagerly besides metrics, I would rather not in Python.

Reply:
Removing "eagerly started" from handle to be consistent with other SDKs